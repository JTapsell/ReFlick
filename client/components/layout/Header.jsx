import React from 'react'
import { Link } from 'react-router-dom'
import { Container, Col, Row } from 'react-bootstrap'

export default function Header () {
  return (
    <Container className="header">
      <Row>
        <Col xs={12} md={8} className="title"> REFLICK </Col>
        <Col xs={6} md={4} className="nav">
          <Link to ='/'>Create! </Link>
          <Link to ='/play/1'>Play!</Link>
        </Col>
      </Row>
    </Container>
  )
}
