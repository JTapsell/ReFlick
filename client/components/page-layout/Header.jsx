import React from 'react'
import { Link } from 'react-router-dom'

export default function Header () {
  return (
    <div className="header">
      <h1>Reflick the remaster we've been waiting for</h1>
      <Link to ='/'>Create!</Link>
      <Link to ='/play'>Play!</Link>
    </div>
  )
}
