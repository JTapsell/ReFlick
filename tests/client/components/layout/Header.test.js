import Header from '../../../../client/components/layout/Header'
import { shallow } from 'enzyme'
import React from 'react'

test('test runner is working', () => {
  expect(true).toBeTruthy()
})
let wrapper
beforeEach(() => {
  wrapper = shallow(<Header />)
})

describe('<Header /> is running', () => {
  it('Renders correct', () => {
    expect(wrapper).toMatchSnapshot()
  })
  // it('Renders correct text', () => {
  //   expect(wrapper.find('div').text()).toBe('REFLICK')
  // })
})
