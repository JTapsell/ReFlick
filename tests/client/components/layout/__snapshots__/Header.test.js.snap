// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> is running Renders correct 1`] = `ShallowWrapper {}`;
